anthropic/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "53fe7c3e-57eb-42ca-8e43-ec92c04b6160",
+  "app_id": "anthropic",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
```

Comment:
@barieom are there any docs/blog posts you'd want to link here to display on the tile? Might as well include this here as well as screenshots to linked dashboards like we have for OpenAi https://app.datadoghq.com/integrations/openai?

Reply:
cc @yahya-mouman

Reply:
For now we don't have any specific blog posts or dashboards for Anthropic. We only have the dedicated LLM overview dashboard. That I already linked in the dashboards list.

Reply:
sounds good, If we can add screenshots of it on the overview tab, that would be great.

Reply:
I created a ticket to do this later when we also have other blogs for Gemini as well https://datadoghq.atlassian.net/browse/MLOB-1922